builder.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * String validator builder function
+ * Allow to build a template for string validations
+ *
+ * Author: Theekshana Wijesinghe
+ */
+
+const StringValidator = require('./');
+
+// These are the public methods exposed by StringValidator class
+const stringValidatorExposedMethods = ['length', 'max', 'min', 'regex', 'includes', 'allow'];
```

Comment:
Did you check with `Object.getOwnPropertyNames(StringValidator.prototype)` ? It additionally contains `constructor` in the array, but we can remove it.
Otherwise, we might need to update this file for each new function exposed to StringValidator.

Reply:
Good point. I will check on this one.